angular.json

```diff
@@ -127,6 +127,7 @@
             "karmaConfig": "src/karma.conf.js",
             "codeCoverage": true,
             "codeCoverageExclude": [
+              "src/app/features/model-3d-viewer/helper.ts",
```

Comment:
I'd prefer `/* istanbul ignore file */` inside the file instead of this

Reply:
done